examples/node-event-hooks/src/main.rs

```diff
@@ -10,10 +10,10 @@
 //! This launch the regular reth node and also print:
 //!
 //! > "All components initialized"
-//! once all components have been initialized and
+//! > once all components have been initialized and
```

Comment:
this is wrong

Reply:
what is correct?

Reply:
I guess there should be a new line?
@shekhirin